libultraship/libultraship/Archive.h

```diff
@@ -21,14 +22,15 @@ namespace Ship
 	{
 	public:
 		Archive(const std::string& MainPath, bool enableWriting);
-		Archive(const std::string& MainPath, const std::string& PatchesPath, bool enableWriting, bool genCRCMap = true);
-		~Archive();
+        Archive(const std::string &MainPath, const std::string &PatchesPath, const std::unordered_set<uint32_t> &ValidHashes, bool enableWriting, bool genCRCMap = true);
+        Archive(const std::vector<std::string> &OTRFiles, const std::string &PatchesPath, const std::unordered_set<uint32_t> &ValidHashes, bool enableWriting, bool genCRCMap = true);
```

Comment:
This probably no longer needs patches path.

Reply:
Would we not still want mods to be imported from the `/mods` folder? If so I think we would still need that, unless we want to hardcode the presence of a `/mods` folder?

Reply:
It just seems weird that we have a vector of OTRs and then a mods folder. I think the /mods folder should be something done in SoH at this point rather than LUS.

Reply:
Oh you only want Patches Path removed from the one that gets the vector of OTRs? That makes much more sense, I'll fix that shortly.

Reply:
Yeah it makes sense to only send LUS a list of archives and let SOH itself deal with the patches path and what archives and in what order to send LUS.

Reply:
Should be handled now.